common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -829,6 +829,7 @@ def verify_teams_discussion_permissions(self, should_have_permission):
         self.assertEqual(self.team_page.discussion_id, self.teams[0]['discussion_topic_id'])
         discussion = self.team_page.discussion_page
         self.assertTrue(discussion.is_browser_on_page())
+        discussion.wait_for_element_presence(".discussion", "Waiting on discussion element", 30)
```

Comment:
I have adopted the style of putting the waits and CSS all in the page files instead of the test files (so I would create a method called wait_for_discussion_presence in the discussion page class). @jzoldak what do you think of that?